.github/workflows/regression-report.yml

```diff
@@ -115,11 +115,11 @@ jobs:
           git checkout -b ${{ steps.get_pr_details.outputs.branch }} \
                forked/${{ steps.get_pr_details.outputs.branch }}
 
-      - name: Checkout contribution repository
+      - name: Checkout test-configs repository
```

Comment:
What for we need this repository checkout?

Reply:
might be same resolved by discussion at https://github.com/checkstyle/checkstyle/pull/15666/files#r1763166880

Reply:
we need the checkout because we need checkstyle-tester folder inside test-configs where java jar would find the patch diff tool jar.

Reply:
Jar can we located at any place.
Lets add new parameter to diff tool to specify path to jar.

Reply:
Or download jar to path we diff tool expects

Reply:
> Jar can we located at any place. Lets add new parameter to diff tool to specify path to jar.

parameter for path diff too jar?

Reply:
> Or download jar to path we diff tool expects

yes, we are downloading that. but, won't we require the cd to move to that place for java jar to execute it?

Reply:
Why we need checkout of whole repository?

Reply:
please remove it, and show me error. We should not do checkout of repository just to create folders that our diff tool expects, we can create folders by "mkdir" if so required.

Reply:
>yes, we are downloading that. but, won't we require the cd to move to that place for java jar to execute it?

download it with path specified, so all folders that required will be created during download action.

Reply:
> parameter for path diff too jar?

in better world it should be `java -jar "$DIFFTOOL_JAR"  --patch-diff-report-tool-path .ci-temp/test-configs/checkstyle-tester/patch-diff-report-tool-0.1-SNAPSHOT-jar-with-dependencies.jar -r "$REPO" -p "$PR_BRANCH" -c "$NEW_MODULE_CONFIG" -m single ..... `

so all required elements are defined by parameters.
Or to be placed in same folder as `"$DIFFTOOL_JAR"` is located, so `--patch-diff-report-tool-path` would have default value as "." (current folder).

Reply:
oh,
am I right that you need this repo just to get this folder and this file - https://github.com/checkstyle/test-configs/blob/main/checkstyle-tester/pom.xml
if yes, lets download it by curl.
In future we might move this file to checkstyle repo, as this config is part of workflow configuration files, or java-diff-tool will keep it as resource and create this file during execution, so will not need to deal with it at all.

Reply:
done

Reply:
Improvement moved to later time https://github.com/checkstyle/test-configs/issues/190